packages/backend/src/apps/paysg/actions/create-payment/index.ts

```diff
@@ -3,8 +3,7 @@ import type { IRawAction } from '@plumber/types'
 import { ZodError } from 'zod'
 import { fromZodError } from 'zod-validation-error'
 
-import { VALIDATION_ERROR_SOLUTION } from '@/apps/postman/common/constants'
```

Comment:
Can we consolidate postman to use the `GenericSolution` as well?

Reply:
Wait postman doesn't even use this. wth. I wonder why this was in there in the first place. Anyway I deleted the constants file from postman.